src/proto/FunctionRpc.proto

```diff
@@ -68,6 +68,9 @@ message StreamingMessage {
     // Worker logs a message back to the host
     RpcLog rpc_log = 2;
 
+    // Worker logs a metric back to the host
+    RpcMetric rpc_metric = 3;
```

Comment:
Instead of defining a new message - Can we reuse RpcLog with Metric category  as this  just another type of log:

https://github.com/Azure/azure-functions-language-worker-protobuf/blob/746863e158795744f8a37f922d448c7499b4a1fc/src/proto/FunctionRpc.proto#L483-L486

As of now we have User logs, System Logs - we can expand and include Metric

@brettsam - thoughts?

Reply:
⏲️

Reply:
My recommendation is to use existing message and introduce new category !

Reply:
Looking deeper into this I'm not sure it's the "best" way to do this. I'd basically end up having to overload the `properties` piece w/ Metric details and then pull it back out on the receiving end. Does this feel right/better over a new type?